docs/interoperability.md

```diff
@@ -106,53 +108,50 @@ is a good reference for enabling CSS modules prior to `2.0.0`.
 
 ```css
 /* Button.module.css */
-.bg { /* public class name in Moonstone/Button */
+.bg { /* public class name in Sandstone/Button */
     background-color: #a4939d !important;
 }
 ```
 
 ```js
 // App.js
-import Button from '@enact/moonstone/Button';
-import { MoonstoneDecorator } from '@enact/moonstone/MoonstoneDecorator';
-import { Component } from 'react';
+import Button from '@enact/sandstone/Button';
+import ThemeDecorator from '@enact/sandstone/ThemeDecorator';
 
 import buttonCss from './Button.module.css';
 
-class App extends Component {
-	render() {
-		return (
-			<div>
-				<Button css={buttonCss}>Click me</Button>
-			</div>
-		);
-	};
+function App() {
+	return (
+		<div>
+			<Button css={buttonCss}>Click me</Button>
+		</div>
+	);
 }
 
-export default MoonstoneDecorator(App);
+export default ThemeDecorator(App);
 ```
 
 ### Internationalization Outside of Enact
 
 #### CRA Example
-In order to use the [`@enact/i18n`](../../developer-guide/i18n) library for internationalization, you can [eject](https://facebook.github.io/create-react-app/docs/available-scripts#npm-run-eject)
+In order to use the [`@enact/i18n`](../../developer-guide/i18n) library for internationalization, you can [eject](https://create-react-app.dev/docs/available-scripts/#npm-run-eject)
 your CRA app, install your required Enact libraries (plus, `@enact/dev-utils`), and update the webpack configuration.
 
 ```json
 // package.json
-
 ...
 "devDependencies" : {
-  "@enact/dev-utils": "2.0.0"
+  "@enact/dev-utils": "^4.1.1"
 },
 "dependencies": {
   ...
-  "@enact/i18n": "2.3.0",
-  "@enact/moonstone": "2.3.0",
+  "@enact/i18n": "^4.0.2",
+  "@enact/sandstone": "^1.5.1",
```

Comment:
```suggestion
  "@enact/sandstone": "^2.0.0",
```

Reply:
It's fixed.
I agree it would be better to use 2.0.0 for later.